content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
+    - Read all questions at the same rate of speed with the ability to stop reading the question at the exact moment a jump is made.
+- Keep the quiz moving at a consistent pace.  A quiz should not be rushed, but it is important to maintain the quickest pace which does not interfere with accuracy.
+- Be open to a challenge and be willing to evaluate whatever the captains or co- captains feel is in error.
+- Facilitate a positive, uplifting environment throughout the quiz.  Quizmasters should speak clearly and be enthusiastic about quizzing. To this end, quizzers should be called by name, the score should be announced often (when a scoreboard is not present), and bonuses and quiz-outs should be promptly recognized.
+
+The Quizmaster must:
+- Identify the question type of each question and begin every question with, “question number_____, question”.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary. 
+    - When an answer is in question (different than what is on the card), the quizmaster will discuss with the answer judge.
+- During the 30-second answering period use the following prompts:
+    - If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer, the quizmaster will ask the quizzer to identify the specific pronoun, (example: “Can you identify ‘He’?” or “Can you identify ‘They’?”) The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification.   A pronoun may need to be identified only if the identification (antecedent) is in context.
+    - On a Chapter-Verse Reference Question (CVR) or Chapter-Only Reference Question (CR), if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+    - On a Finish-The-Verse (FTV), Quote, or Finish This (FT) type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again."  This will occur as often as needed until the quizzer quotes the verse perfectly or the 30-second time limit expires.  
+        - The quizmaster is not allowed to say “again” until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+    - Once all of the information in both the question and answer have been given, but something needs to be fixed (e.g. Unique word) the quizmaster will say “Again”.
+    - On a Situation (SIT) type question, when the quizzer has completed the Quotation the quizmaster will say, “Quote is complete.”
+    - On an INT or MA Question, if all of the information in the answer has been given, but the quizzer is missing information in the question, the quizmaster will say “More”.
```

Comment:
I REALLY want to change this. The line between "Question" and "Answer" is pretty gray, when a quizzer jumps before the end of the Question.

Reply:
I'd be OK with more narrowly defining "question". It can mean:

- The whole of the question, including the answer portion. Also including the reference and type.
- The full question portion of the whole "question"
- The content the quizmaster said

I think we should come up with 3 distinct words for the above items, define them, and be consistent with them. The word "question" is seriously overloaded.

Reply:
a stab:
Required Content
Question [portion of Required Content]
Read Question

Reply:
Another stab attempt:

- **"Question"** = Everything (type, ref, question text, answer text, etc.)
- **"Answer"** = Everything the quizzer says (answer, remaining question, ref, etc.)
- **"Question Text"** = The full question text of the question
- **"Answer Text"** = The full answer text of the question
- **"Read Question"** = Everything the QM says (excluding prompts and rulings)

I don't like having "Question" and "Question Text", but we're not going to be able to stop people from using the word question when we QMs say it 117 times every quiz meet. So I like explicitly defining it to mean everything.

Reply:
I like your stab. If adopted, I'll need to change some portion where I used the word "response"

Reply:
Since you're doing most of the writing thus far, you get to decide if it's adopted. :)

Reply:
I'm excited to implement it. But I think a lot of choices will need to be made. The rule book is littered with imprecise uses of those 5 terms. So in cases where none of the 5 exactly maintain FE, a choice has to be made.

Reply:
Yep. And I think between the 4 of us, we'll get it effectively FE.